src/plots/gl2d/camera.js

```diff
@@ -131,7 +131,7 @@ function createCamera(scene) {
 
                         if(Math.abs(dx * dydx) > Math.abs(dy)) {
                             result.boxEnd[1] = result.boxStart[1] +
-                                Math.abs(dx) * dydx * (Math.sign(dy) || 1);
+                                Math.abs(dx) * dydx * (dy >= 0 ? 1 : -1);
```

Comment:
[Non-blocking comment] It might be worth making a `Lib.sign` method at some point.